ood-portal-generator/lib/ood_portal_generator/view.rb

```diff
@@ -38,7 +38,7 @@ def initialize(opts = {})
       @maintenance_ip_whitelist = Array(opts.fetch(:maintenance_ip_whitelist, []))
 
       # Security configuration
-      @security_disable_frames = opts.fetch(:security_disable_frames, true)
+      @security_csp_frame_ancestors = opts.fetch(:security_csp_frame_ancestors, "#{@protocol}#{@servername ?  @servername : OodPortalGenerator.fqdn}")
```

Comment:
I know this is pulled in now, but should this have accounted for `proxy_server` too?  Will this work with HA deployments who have a proxy?

Reply:
Nevermind, it defaults to `servername` but folks with proxy servers can configure they're proxy here. Disregard!